packages/app/src/app.tsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import { Main } from '@sample-monorepo/components';
+import {hot} from 'react-hot-loader/root'
```

Comment:
run `yarn lint --fix` in root to fix all missing semi-colons automatically. (PR fails linting in CI)